crates/crypto/src/key/private_key.rs

```diff
@@ -0,0 +1,59 @@
+use crate::key::{Key, KeyError, PublicKey};
+use ssi_jwk::JWK;
+use ssi_jws::sign_bytes;
+
+#[derive(Clone, PartialEq, Debug)]
+pub struct PrivateKey(pub(crate) JWK);
+
+impl PrivateKey {
+    /// Derive a [`PublicKey`] from the target [`PrivateKey`].
+    pub fn to_public(&self) -> PublicKey {
+        PublicKey(self.0.to_public())
```

Comment:
can you explain the `.0.` part? after installing some vscode rust extensions, it tells me this 0 is equivalent to JWK (which i guess is the 0th param passed into the struct on line 6 above), is that correct? and if it is, is calling `self.0.foo()` different than  calling `self.jwk().foo()` ?